static/css/frame.scss

```diff
@@ -492,6 +528,7 @@ body {
   height: 40px;
   border-radius: 3px;
   position: absolute;
+  left: 170.5px;
```

Comment:
The color picker look more centered with those numbers.

Reply:
Yeah, it's true. I guess subpixel rounding should be fine, since we're only supporting Firefox, and the problem starts with the `.color-board .triangle` having `left: 9.5px`, and that's not part of this patch. Still seems odd to me, but that's fine.

Reply:
I was surprised to be able to discern the difference when I tried to changed it to a whole number while working on it.